Applications/Spire/Include/Spire/Charting/ColorPicker.hpp

```diff
@@ -21,9 +24,17 @@ namespace Spire {
       */
       ColorPicker(int width, int height, QWidget* parent = nullptr);
 
+      //! Calls the provided slot when the color preview signal is triggered.
+      /*
```

Comment:
Fix up all of these ```/*``` to ```/*!```

Reply:
Do you want me to fix all of them across the entire solution?